containers/podman-desktop
# chore: spinner alignment

### What does this PR do?

We have a simple svg Spinner component used in 1 place (PreferencesConnectionCreationRendering), and a different PatternFly spinner used in 18 places, where the same 7 lines of code with 6 PF classes is copy/pasted.

I checked with Mairin and she prefers the PF spinner more, at least for now. So this change:
- replaces our existing Spinner with the PF spinner code
- adds a size property to allow the one existing use to be larger and closer to the existing SVG size
- uses the Spinner to replace all the existing uses

### Screenshot/screencast of this PR

N/A. Button spinners unchanged; one existing spinner close enough and used so fast you can't tell the difference.

### What issues does this PR fix or reference?

N/A

### How to test this PR?

Create a new Podman machine. You'll see the existing spinner very briefly loading the page, and the button spinner after clicking Create.
